src/shapes/Text/StyledText.ts

```diff
@@ -246,14 +243,18 @@ export abstract class StyledText<
   }
 
   /**
-   * get the reference, not a clone, of the style object for a given character
+   * get the reference, not a clone, of the style object for a given character,
+   * if not style is set for a pre det
    * @param {Number} lineIndex
    * @param {Number} charIndex
-   * @return {Object} style object
+   * @return {Object} style object a REFERENCE to the existing one or a new empty object
    */
-  _getStyleDeclaration(lineIndex: number, charIndex: number) {
+  _getStyleDeclaration(
+    lineIndex: number,
+    charIndex: number
+  ): TextStyleDeclaration {
     const lineStyle = this.styles && this.styles[lineIndex];
-    return lineStyle ? lineStyle[charIndex] : null;
+    return lineStyle ? lineStyle[charIndex] ?? {} : {};
```

Comment:
Are we ok with abbreviations now or not yet?
```suggestion
    return lineStyle?.[charIndex] ?? {};
```

Reply:
is so much longer the abbreviation because safari.

Reply:
i m also not sure about performance in general of that abbrevieation